packages/squidex/schema/crn/schemas/teams.json

```diff
@@ -46,6 +46,38 @@
           "isHalfWidth": true
         }
       },
+      {
```

Comment:
im not so sure about adding these two fields like this, especially without scripting. So this would mean that the user would have to change the team status manually and remember to record the date in another field when they do that, right? I think we should just use the same mechanism we used to alumnis which is that we only have a single field with a date. We would make an assumption that this would only ever be a past date which would mean that to check if the team is active we would just check the present of any date in that field

Reply:
ok now i see you've added the script in this PR https://github.com/yldio/asap-hub/pull/2106

See - i still think it should just be a single field

Reply:
it makes sense :) I'll change the https://github.com/yldio/asap-hub/pull/2106 to have a single field. I'll close this one and update everything in https://github.com/yldio/asap-hub/pull/2106 then. I was trying to break in small PRs to make them easier to review, but I think it got messier